ui/src/App.scss

```diff
@@ -195,7 +195,11 @@ body {
 
 .ant-modal-wrap {
   padding-top: $layout-header-height;
-  z-index: $zindex-header + 1 !important;
+  z-index: $zindex-footer + 1 !important;
```

Comment:
shall we move all these to `styleVariables` so that we know what is on top of what?